types/chart.js/index.d.ts

```diff
@@ -858,7 +858,7 @@ declare namespace Chart {
         resize?(chartInstance: Chart, newChartSize: ChartSize, options?: any): void;
         destroy?(chartInstance: Chart): void;
 
-        /** @deprecated since version 2.5.0. Use `afterLayout` instead. */
+        /** Deprecated since version 2.5.0. Use `afterLayout` instead. */
```

Comment:
Kept getting the following error when running `npm run lint chart.js` until I removed the `@` symbol.

```
The 'no-redundant-jsdoc-2' rule threw an error in '/Users/steve/github/DefinitelyTyped/types/chart.js/index.d.ts':
Error: Unexpected tag kind: JSDocDeprecatedTag
```